hms-app/src/pages/Presentations.tsx

```diff
@@ -0,0 +1,203 @@
+                    </Badge>
+                  </Group>
+                </Card.Section>
+              </Grid.Col>
+            </Grid>
+          </Card.Section>
+
+          <Card.Section style={{ height: '20%' }} className={classes.section}>
+            <Text className={classes.label}>Problem</Text>
+            <Text
+              mt="sm"
+              style={{
+                backgroundColor: 'white',
+                height: '15vh',
+              }}
+            >
+              {idea.problem}
+            </Text>
+          </Card.Section>
+
+          <Card.Section style={{ height: '20%' }} className={classes.section}>
+            <Text className={classes.label}>Goal</Text>
+            <Text mt="sm" style={{ backgroundColor: 'white', height: '15vh' }}>
+              {idea.goal}
+            </Text>
+          </Card.Section>
+
+          <Card.Section style={{ height: '20%' }} className={classes.section}>
+            <Grid>
+              <Grid.Col span={6}>
+                <Card.Section>
+                  <Text className={classes.label}>Skills</Text>
+                  <Text
+                    mt="sm"
+                    style={{
+                      backgroundColor: 'white',
+                      height: '15vh',
+                    }}
+                  >
+                    <ul>
+                      {idea.requiredSkills?.map((skill, index) => (
+                        <li>{skill.name}</li>
+                      ))}
+                    </ul>
+                  </Text>
+                </Card.Section>
+              </Grid.Col>
+
+              <Grid.Col span={6}>
+                <Card.Section>
+                  <Text className={classes.label}>Participants</Text>
+                  <Text
+                    mt="sm"
+                    style={{
+                      backgroundColor: 'white',
+                      height: '15vh',
+                      border: '1px solid transparent',
+                    }}
+                  >
+                    <Grid>
+                      {idea.participants?.map((participant, index) => (
+                        <Grid.Col span={4}>
+                          <div
+                            style={{
+                              display: 'flex',
+                              alignItems: 'center',
+                              gap: '10px',
+                            }}
+                          >
+                            <Avatar
+                              color="indigo"
+                              radius="xl"
+                              size="md"
+                              src={
+                                'https://avatars.githubusercontent.com/u/10353856?s=460&u=88394dfd67727327c1f7670a1764dc38a8a24831&v=4'
+                              }
+                            />
+                            {participant.user.firstName}{' '}
+                            {participant.user.lastName}
```

Comment:
maybe this should get converted into a function as part of a `user` class (or some function that accepts a user as input).  Then it would do something a bit like:

```javascript
function renderName(user: User): string {
  user.firstName + (user.lastName ? ' ' + user.lastName : '');
}
```

Reply:
this just prevents an extra ` ` (space) from being included when the lastName is null/undefined